src/CurrencyTransformer/EnglishCurrencyTransformer.php

```diff
@@ -31,8 +31,8 @@ public function toWords($amount, $currency, $options = null)
             ->useRegularExponents($exponentInflector)
             ->build();
 
-        $decimal = (int) ($amount / 100);
-        $fraction = abs($amount % 100);
+        $decimal = $currency != 'UGX' ? (int) ($amount / 100) : (int) ($amount);
+        $fraction = $currency != 'UGX' ? abs($amount % 100) : abs($amount % 1);
```

Comment:
Is there any chance it could be more generic (not tight to `UGX` currency)? E.g. adding parameter that controls if currency has a fraction part.

Reply:
I believe so as nothing can fail code